src/server.c

```diff
@@ -6000,6 +6003,11 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
                     "master_link_down_since_seconds:%jd\r\n",
                     server.repl_down_since ?
                     (intmax_t)(server.unixtime-server.repl_down_since) : -1);
+            } else {
+                info = sdscatprintf(info,
+                    "master_link_up_since_seconds:%jd\r\n",
+                    server.repl_up_since ? // defensive code, should never be 0 when connected
```

Comment:
line comment (`//`)? :stuck_out_tongue:

Reply:
it is in a development state, the comment is relatively casual. anyway, fixed 😄